lib/Review.php

```diff
@@ -22,12 +22,25 @@
  *
  * @package Stripe
  */
-class Review extends \Stripe\ApiResource
+class Review extends ApiResource
 {
     const OBJECT_NAME = "review";
 
-    use \Stripe\ApiOperations\All;
-    use \Stripe\ApiOperations\Retrieve;
+    use ApiOperations\All;
+    use ApiOperations\Retrieve;
+
+    /**
```

Comment:
I put all of the review reasons into a single list of reasons, even though they don't all apply to all of the reason types (open, close and just reason). It didn't make sense to me to create 3 separate reason sections.